src/site/content/en/blog/sanitizer/index.md

```diff
@@ -78,32 +78,14 @@ const $div = document.querySelector('div')
 const user_input = `<em>hello world</em><img src="" onerror=alert(0)>`
 const sanitizer = new Sanitizer()
 $div.setHTML(user_input, sanitizer) // <div><em>hello world</em><img src=""></div>
-
 ```
 
 It is worth noting that `setHTML()` is defined on [`Element`](https://developer.mozilla.org/docs/Web/API/Element). Being a method of `Element`, the context to parse is self-explanatory (`<div>` in this case), the parsing is done once internally, and the result is directly expanded into the DOM.
 
-If you don't want to expand directly into the DOM, you can also get the result as an `HTMLElement`.
-
+To get the result of sanitization as a string, you can use `.innerHTML` from the `setHTML()` results.
 
 ```js
-const user_input = `<em>hello world</em><img src="" onerror=alert(0)>`
-const sanitizer = new Sanitizer()
-sanitizer.sanitizeFor("div", user_input) // HTMLDivElement <div>
-```
-
-If you want to get the result of sanitization as a string, you can use `.innerHTML` from the `HTMLElement` you got here. (You still need to specify the appropriate context to parse.)
-
-
-```js
-sanitizer.sanitizeFor("div", user_input).innerHTML // <em>hello world</em><img src="">
-```
-
-If you already have a user-controlled `DocumentFragment` and you want to remove only the harmful stuff from it, use `sanitize()`.
-
-
-```js
-$div.replaceChildren(s.sanitize($userDiv));
+$div.setHTML(user_input, sanitizer).innerHTML // <em>hello world</em><img src="">
```

Comment:
Hah. You still have to tell us what user_input is. I think we need to keep that line for this example to make sense. :-)